resources/lib/view.py

```diff
@@ -6,15 +6,34 @@
 
 
 def build_categories(lang):
-    categories = [mapper.map_categories_item(
-        item) for item in api.categories(lang)]
+    categories = []
+    categories.append(mapper.create_newest_item())
```

Comment:
you can directly init category with the first menu items instead of using `append`

```python
category = [
  mapper.create_newest_item(),
  mapper.create_most_viewed_item(),
  mapper.create_last_chance_item()
]
```